pkg/Microsoft.Private.PackageBaseline/packageIndex.json

```diff
@@ -1196,7 +1196,7 @@
         "net461": "4.1.0.0",
         "monoandroid10": "Any",
         "monotouch10": "Any",
-        "uap10.0.16300": "4.3.1.0",
+        "uap10.0.16300": "4.4.0.0",
```

Comment:
Hasn't this version of UAP already shipped?  If so then you can't claim it had this new version inbox.  You also need to make sure the package carries an implementation for UAP (since you increased the ref assembly version of the netstandard2 ref).  /cc @joperezr

Reply:
What changes would be required for that?

Reply:
Undo this portion of the change and move https://github.com/dotnet/corefx/blob/master/src/System.Data.SqlClient/src/Configurations.props#L14 to PackageConfigurations.  You'd also need to remove https://github.com/dotnet/corefx/blob/master/src/System.Data.SqlClient/pkg/System.Data.SqlClient.pkgproj#L9-L12

Reply:
> You also need to make sure the package carries an implementation for UAP

Do we also need to change the implementation for UAP to be built against the shipped UAP, and not against the live UAP that it is building against right now?

Reply:
I had discussed this before with @joperezr.  Typically we move the version-less identifier to mean v.next and add support for targeting the last versioned identifier.  For libraries that ship as packages we try to keep them targeting the minimal version so that we don't get cross-compile explosion.

Reply:
uap10.0.16300 hasn't shipped yet, this is the place holder for uap v-next so I believe that increasing the version on the package index should be fine.

Reply:
@ericstj Since uap10.0.16300 hasn't shipped yet, are these changes good to merge as is, or should I still include your recommended configuration changes?

Reply:
@joperezr did this version of [UAP already ship a version of SqlClient inbox](https://github.com/dotnet/corefx/blob/release/uwp6.0/pkg/Microsoft.Private.PackageBaseline/packageIndex.json#L1138)?  If so then the same applies, it just means that our index is out of sync with what's been released...

Reply:
Yes you are right, it was shipped then, I'm preparing a PR right now on master that will update this packageIndex to reflect what we shipped as part of that metapackage

Reply:
@joperezr Is this version bump for uap10.0.16300 fine, or does it need changes to account for the uap versions in your PR? https://github.com/dotnet/corefx/pull/27385

Reply:
I believe that what @ericstj is suggesting as that we might need to target uap10.0.16299 instead. You won't be able to do that today since we don't really build a targeting pack for it. @ericstj given that this is master, what would you recommend? @corivera do we want these changes to flow to 2.1 preview 2?

Reply:
@joperezr We already did a version bump a while back when adding new APIs, so I'm not sure how essential it is to bump it again to 4.4.0. 

@weshaggard Are there any issues with just leaving this as version 4.3.1 for preview2?

Reply:
Yes it should be but we need to make sure we to address this before we ship 2.1 RTM otherwise this package may cause some issues.

Reply:
> we might need to target uap10.0.16299 instead.

Exactly, since you are adding API to the netstandard 2.0 ref and UAP 10.0.16299 supports netstandard2.0 you have to update the implementation by including the UAP 10.0.16299 build in the package.

I would just wait to do this until after @joperezr's change (or even as part of it).  You can do that in release or wait till his changes make it to master.  Either way you need to make sure this change gets into release/2.1.

Reply:
@corivera just as FYI my change is now merged.